CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.16.0
+
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add `clearable` behavior to the Pickers (#9095) @noraleonte
+
+The Pickers now have an out-of-the box implementation for clearing the field value. You can see the documentation for this behavior on the [Date Picker documentation](https://mui.com/x/react-date-pickers/date-picker/#clearing-the-value).
```

Comment:
These are the conventions we've established: https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800

Essentially, your choices are:

- "pickers" if referring to the general concept of the thing
- "Pickers" if referring to our products specifically
- `<WhateverPicker />` if you need to make it clear which component you're referring to
  - this [Base UI Tabs doc](https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800) is a good example of when to employ this style:
  <img width="723" alt="Screenshot 2023-09-29 at 10 55 53 AM" src="https://github.com/mui/mui-x/assets/71297412/8aca7f7b-27aa-46da-80b7-46351022e670">


So in this case specifically, I'd argue it should be "Pickers and Fields" to make it clear that we're referring to the products. These are proper nouns, and should always be capitalized. I would only use lowercase "pickers" if we were referring to the general concept (which would probably never happen in something like a changelog).

Reply:
Think about it like this: it would be weird if I wrote "Sam Sycamore" when using my full name but "sam" when I used just my first name.

Reply:
Thanks for the clarification @samuelsycamore ... for this release it it's too late, but for all future writings (not only release docs/changelogs) this is valuable insight 

Highly appreciated! 🙇🏼‍♂️

Reply:
No big deal here, but consistency is key! I'm always happy to help try to figure the proper usage when it's not clear.

Reply:
> for this release it it's too late, but for all future writings (not only release docs/changelogs) this is valuable insight

But is it too late? 🤔 
Yes, a portion of people might have already seen the initial version of the changelog, but it can be adjusted in both GitHub as well as our master branch. 👌

Reply:
Well overall I'd say it's a pretty minor thing here and maybe not even worth the effort unless you were bulk editing (find & replace) a bunch of documents all at once. But definitely important to be meticulous about style conventions in the docs—I think it contributes a lot to the feeling of professionalism.